README.md

```diff
@@ -38,3 +38,13 @@ Same applies to output files - when not specified, `cmorse` will write to `stdou
  - Clone git repository - `git clone https://github.com/Jacajack/cmorse.git && cd cmorse`
  - Invoke makefile - `make all`
  - To install use `sudo make install`, and to uninstall - `sudo make uninstall`.
+
+## Developing cmorse with Docker
+ - Clone git repository - `git clone https://github.com/Jacajack/cmorse.    git && cd cmorse`
+ - Build docker image - `docker build -t cmorse .`
+ - Run `cmorse` binary using docker container based on the cmorse image
+```
+docker run -it cmorse cmorse
```

Comment:
Also, I think it would look more nicely if you left only the run command, without the "Hello, World!" thing after

 - Run `cmorse` binary using docker container based on the cmorse image - `docker run -it cmorse cmorse`